micromamba/tests/dump_proxy_connections.py

```diff
@@ -0,0 +1,20 @@
+from mitmproxy import ctx
```

Comment:
This is great. Can you add a tiny description what this does and how to use it? Might be useful even outside of the tests.